docs/design/lexical_conventions/words.md

```diff
@@ -87,6 +91,7 @@ The following words are interpreted as keywords:
 ## Alternatives considered
 
 -   [Character encoding: We could restrict words to ASCII.](/proposals/p0142.md#character-encoding-1)
+-   [Normalization forms: We could require a different normalization form.](/proposals/p0142.md#normalization-forms)
```

Comment:
That section discusses several alternatives besides a different normalization form.

```suggestion
-   [Normalization form alternatives considered](/proposals/p0142.md#normalization-forms)
```

Reply:
I have updated it!